app/routes/bestuurseenheid/reglementen/index.js

```diff
@@ -0,0 +1,57 @@
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+
+const PAGE_SIZE = 10;
+
+export default class ReglementenIndexRoute extends Route {
+  @service store;
+  @service fastboot;
+
+  queryParams = {
+    page: {
+      refreshModel: true,
+    },
+    to: {
+      refreshModel: false,
+    },
+    from: {
+      refreshModel: false,
+    },
+  };
+
+  async model(params) {
+    const bestuurseenheid = this.modelFor('bestuurseenheid');
+    const model = await this.store.query('uittreksel', {
+      include: ['zitting', 'behandeling-van-agendapunt.besluiten'].join(),
+      'filter[zitting][bestuursorgaan][is-tijdsspecialisatie-van][bestuurseenheid][:id:]':
+        bestuurseenheid.id,
+      'fields[published-resource]': 'id',
+      'fields[zitting]': 'id',
+      'fields[besluit]': 'id',
+      sort: '-zitting.geplande-start',
+      page: {
+        number: params.page || 0,
+        size: PAGE_SIZE,
+      },
+    });
+    const pageNumber = Number(params.page) || 0;
+    model.meta.page = pageNumber;
```

Comment:
query params are supposed to be defined on the controller:
https://guides.emberjs.com/v4.8.0/routing/query-params/

what's the reason for tacking them onto the ember data model?

Reply:
actually hold off on this, the docs may be wrong

Reply:
hmm based on you comment below you told me to move the logic of fetchi the extracts from controller to model hook, on the route(or maybe I misunderstood that ?). This is the main reason